docs/src/main/sphinx/connector/jdbc-procedures.fragment

```diff
@@ -10,3 +10,13 @@ Procedures
 
       USE example.myschema;
       CALL system.flush_metadata_cache();
+
+* ``system.pass_through('statement')``
```

Comment:
we have `system.query`  table function.

what should be most consistent name for this?

Reply:
Would it make sense to rename it also to `query`, or would it confuse users that there is a `DDL query` and a `table query`?

Can it even be the same name?

Reply:
> Can it even be the same name?

i think the procedures and table functions do not share a namespace today, but i don't know what's the long term intention. cc @martint @dain @kasiafi

Reply:
how about `system.update` to match JDBC update statement, and that table function should return number of affected rows.